app/src/main/res/layout/profile_chooser_add_view.xml

```diff
@@ -15,28 +15,39 @@
       type="androidx.databinding.ObservableField&lt;Boolean&gt;" />
   </data>
 
-  <LinearLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:gravity="center_horizontal"
-    android:orientation="vertical">
+    android:paddingStart="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_constraint_layout_padding_start_profile_added : @dimen/profile_chooser_add_view_constraint_layout_padding_start_profile_not_added}"
+    android:paddingEnd="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_constraint_layout_padding_end_profile_added : @dimen/profile_chooser_add_view_constraint_layout_padding_end_profile_not_added}"
+    app:layoutMarginTop="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_constraint_layout_margin_top_profile_added : @dimen/profile_chooser_add_view_constraint_layout_margin_top_profile_not_added}">
 
     <LinearLayout
       android:id="@+id/add_profile_item"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:gravity="@{hasProfileEverBeenAddedValue ? Gravity.CENTER_HORIZONTAL : Gravity.CENTER_VERTICAL}"
-      android:orientation="@{hasProfileEverBeenAddedValue ? LinearLayout.VERTICAL : LinearLayout.HORIZONTAL}"
-      app:layoutMarginEnd="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_margin_end_profile_already_added : @dimen/profile_view_not_added_margin}"
-      app:layoutMarginStart="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_margin_start_profile_already_added : @dimen/profile_view_not_added_margin}"
-      app:layoutMarginTop="@{hasProfileEverBeenAddedValue ? @dimen/space_0dp : @dimen/profile_chooser_add_view_margin_top_profile_not_added}">
-      
+      android:gravity="@{hasProfileEverBeenAddedValue ? @integer/profile_chooser_add_view_add_profile_item_gravity_profile_added : @integer/profile_chooser_add_view_add_profile_item_gravity_profile_not_added}"
+      android:orientation="@{hasProfileEverBeenAddedValue ? @integer/profile_chooser_add_view_add_profile_item_orientation_profile_added : @integer/profile_chooser_add_view_add_profile_item_orientation_profile_not_added}"
+      android:layout_marginEnd="@{hasProfileEverBeenAddedValue ? @dimen/profile_chooser_add_view_add_profile_item_android_margin_end_profile_added : @dimen/profile_chooser_add_view_add_profile_item_android_margin_end_profile_not_added}"
```

Comment:
Hey @BenHenning I am getting error upon building the project ig these lines(31 - 33) are the reason behind the error and I think its important to add these lines because the `land\profile_chooser_add_view` has the same reference 
https://github.com/oppia/oppia-android/blob/7653898a16f18c36596058aeefc702fd91331d13/app/src/main/res/layout-land/profile_chooser_add_view.xml#L25
https://github.com/oppia/oppia-android/blob/7653898a16f18c36596058aeefc702fd91331d13/app/src/main/res/layout-land/profile_chooser_add_view.xml#L28

and this is the error
![image](https://user-images.githubusercontent.com/92685493/192163650-bae11528-6e20-4793-a955-43206a321110.png)

C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\build\generated\source\kapt\debug\org\oppia\android\DataBinderMapperImpl.java:135: error: cannot find symbol
import org.oppia.android.databinding.ProfileChooserAddViewBindingImpl;
                                    ^
  symbol:   class ProfileChooserAddViewBindingImpl
  location: package org.oppia.android.databindingC:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\ConstraintLayoutAdapters.java:14: warning: Application namespace for attribute app:layout_constraintEnd_toEndOf will be ignored.
  public static void setConstraintEndToEndOf(@NonNull View view, int constraintToId) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\ConstraintLayoutAdapters.java:24: warning: Application namespace for attribute app:layout_constraintHorizontal_bias will be ignored.
  public static void setHorizontalBias(@NonNull View view, float value) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\DrawableBindingAdapters.java:24: warning: Application namespace for attribute app:roundedRectDrawableWithColor will be ignored.
  public static void setBackgroundDrawable(@NonNull View view, @ColorInt int colorRgb) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\DrawableBindingAdapters.java:33: warning: Application namespace for attribute app:topRoundedRectDrawableWithColor will be ignored.
  public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int colorRgb) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\EditTextBindingAdapters.java:13: warning: Application namespace for attribute app:textChangedListener will be ignored.
  public static void bindTextWatcher(@NonNull EditText editText, TextWatcher textWatcher) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\GuidelineBindingAdapters.java:13: warning: Application namespace for attribute app:layout_constraintGuide_percent will be ignored.
  public static void setGuidelinePercentage(@NonNull Guideline guideline, float percentage) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\GuidelineBindingAdapters.java:22: warning: Application namespace for attribute app:layout_constraintGuide_end will be ignored.
  public static void setConstraintGuidelineEnd(@NonNull Guideline guideline, float guideEndPx) {
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\ImageViewBindingAdapters.java:21: warning: Application namespace for attribute app:srcCompat will be ignored.
  public static void setImageDrawableCompat(
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\ImageViewBindingAdapters.java:29: warning: Application namespace for attribute app:srcCompat will be ignored.
  public static void setImageDrawableCompat(
                     ^C:\Users\akuka\OneDrive\Desktop\opensource\oppia-android\app\src\main\java\org\oppia\android\app\databinding\ImageViewBindingAdapters.java:44: warning: Application namespace for attribute profile:src wi

Reply:
@Akshatkamboj14 usually a binding reference error comes up when the binding is failed to be generated. This is caused by an error in the layout that is stopping the databinding compiler from succeeding, and the error is usually later in the build logs. Could you upload the entire build log to a Gist and link to it?